### PR TITLE
Docs for next version 

### PR DESCRIPTION
**Pull request by martinbonnin**

### CODE REVIEW

File: docs/source/advanced/nullability.mdx [hunk 1]
```diff
@@ -82,270 +88,233 @@ fun User(user: GetUserQuery.User) {
...
+
+(Adding `@catchByDefault(to: NULL)` is a no-op for codegen that unlocks using `@catch` in your operations.)
+
+## Migrate to semantic nullability
+
+Semantic nullability is the most useful for schemas that are nullable by default. These are the schemas that require "handling every field error". 
+
+In order to change that default to "opt-in the errors you want to handle", you can use the following approach:
+
+1. import the nullability directives.
```
Comment (BoD) [inline docs/source/advanced/nullability.mdx:315]:
```suggestion
1. Import the nullability directives.
```

File: docs/source/advanced/nullability.mdx [hunk 2]
```diff
@@ -82,270 +88,233 @@ fun User(user: GetUserQuery.User) {
...
+1. Default to coercing to null: `extend schema @catchByDefault(to: NULL)`. This is a no-op to start exploring the directives.
+1. Add `@catch` to individual fields, get more comfortable with how it works.
+1. When ready to do the big switch, change to `extend schema catch(to: THROW)` and (at the same time) add  `query GetFoo @catch(to: NULL) {}` on all operations/fragments (this is a no-op).
+1. From this moment on, new queries written are `catch(to: THROW)` by default.
```
Comment (BoD) [inline docs/source/advanced/nullability.mdx:319]: I find this sentence not 100% clear - maybe something like:

```suggestion
1. From this moment on, new queries will be `catch(to: THROW)` by default.
```
?

**Review by BoD — APPROVED** [review]
👍

File: README.md [hunk 3]
```diff
@@ -88,15 +88,15 @@ Add the plugin to your `build.gradle.kts`:
 
 ```kotlin
 plugins {
-  id("com.apollographql.apollo3") version "4.0.0-beta.7"
+  id("com.apollographql.apollo") version "4.0.0-beta.7"
```
Comment (BoD) [inline README.md:91]: This reminds me we need to update `release.main.kts`

Comment (martinbonnin) [reply README.md:91]: This was done in https://github.com/apollographql/apollo-kotlin/commit/8d37c6119a43a620be175841050913c246ec1139 (but needs to be tested)

**Pull request merged**